src/helper/test-helper.ts

```diff
@@ -469,15 +477,43 @@ const queryMockResponse = {
     createClaimableBalanceToPublicKey: {
       edges: [],
     },
+    setOptionsByPublicKey: {
+      edges: [],
+    },
   },
 };
 
 export const register = new Prometheus.Registry();
 
+const mercuryErrorCounter = new Prometheus.Counter({
```

Comment:
pulled these out to be constructed here so that we can use them across instance of `MercuryClient` now that we use a second one for the integrity checks.